src/JuliaRunClient.jl

```diff
@@ -254,7 +254,11 @@ macro result(req)
         _server_exception = nothing
         try
             res = $(esc(req))
-            (res["code"] == 0) ? res["data"] : throw(ApiException(res["code"], res["data"], res))
+            if isa(res, Dict)
+                (res["code"] == 0) ? res["data"] : throw(ApiException(res["code"], res["data"], res))
+            else
+                res
```

Comment:
I thought all APIs return JSON. Which/when can something other than that get returned?

Reply:
Yeah, the new API `initializeCluster` returns a `Context` object.